test/gtest/common/test_perf.cc

```diff
@@ -210,9 +210,7 @@ void test_perf::test_params_init(const test_spec &test,
         params.max_iter    = test.iters;
     } else {
         params.warmup_iter = 0;
-        params.max_iter    = ucs_min(20u, test.iters /
-                                          ucs::test_time_multiplier() /
-                                          ucs::test_time_multiplier());
+        params.max_iter    = 1;
```

Comment:
IMO 1 is not enough.. maybe 100?
e.g keep the previous calculation, and add `params.max_iter = ucs_max(params.max_iter, 100)`

Reply:
Why do we need more than one iteration when running with Valgrind?